test/test_playbooks/operating_system.yml

```diff
@@ -5,6 +5,8 @@
   - name: Load server config
     include_vars:
       file: server_vars.yml
+    tags:
+      - always
```

Comment:
Is this used?

Reply:
Those tags are inserted there to solve a delicate problem...
When you hack on the molude and the playbook fails half way, You might have entities left that prevent to meet the expected_change in the next run. With those tags in place, you can run the playbook with `--tags cleanup` and it will delete the stale operating system without listening for the expected change.